config.ini

```diff
@@ -354,6 +354,16 @@ extractor = kconfigs.rpm.RpmExtractor
 index = https://packages.microsoft.com/cbl-mariner/2.0/prod/base/x86_64/
 key = azurelinux
 
+[azl3_preview_x86_64]
+name = Azure Linux
```

Comment:
Let's name this kernel "Azure Linux Preview" as a way to distinguish between preview and prod versions of the kernel. Same with the ARM64 version

Reply:
Done!